src/fee_estimation/mempool_space.rs

```diff
@@ -16,7 +16,7 @@ pub struct MempoolSpaceClient {}
 
 impl MempoolSpaceClient {
     pub async fn fee_rate(priority: TxPriority) -> Result<FeeRate, BriaError> {
-        let url = "https://mempool.space/api/v1/fees/recommended";
+        let url = "http://localhost:8332/api/v1/fees/recommended";
```

Comment:
the port mempool is serving on is 8999 by default.
```suggestion
        let url = "http://localhost:8999/api/v1/fees/recommended";
```
this needs to be exposed by including it the `docker-compose.override.yml`